cookbook/extend-entities.rst

```diff
@@ -76,11 +76,14 @@ Create your own Entity for example in the `src` folder and extends the Entity wi
 Configuration
 -------------
 
-You can specify your new entity and if it exists your repository in the `sulu_security` 
-configuration section in the file app/config/config.yml.
+You can specify your new entity and if it exists your repository the
+configuration section in the file ``config/packages/*``.
```

Comment:
Maybe we should also mention that these files have to be created if they don't exist? Because as far as I see not all of these files exist.